UT-Alignment-12d-1/UT-Alignment-12d-1.ifc

```diff
@@ -0,0 +1,46 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition[Ifc4x3NotAssigned]'), '2;1');
+FILE_NAME ('UT-Alignment-12d-1.ifc', '2021-02-11T13:13:50', ('Lee Gregory'), ('12d Solutions - Ranch'), 'IFC Engine rev 1257', '12d Model 15.0 Alpha 490', '');
+FILE_SCHEMA (('IFC4X3_RC2'));
+ENDSEC;
+DATA;
+#1 = IFCPROJECT('2LJesmETv6W9jWXnI$q1Aj', #2, 'UT-Alignment-12d-1', 'Alignment with left arc in HG  but no VG', $, $, $, (#14), #9);
+#2 = IFCOWNERHISTORY(#3, #8, $, .NOCHANGE., $, $, $, 1613009630);
+#3 = IFCPERSONANDORGANIZATION(#4, #6, $);
+#4 = IFCPERSON($, 'Gregory', 'Lee', $, $, $, (#5), $);
+#5 = IFCACTORROLE(.CIVILENGINEER., 'Designer', $);
+#6 = IFCORGANIZATION($, '12d Solutions - Ranch', $, (#7), $);
+#7 = IFCACTORROLE(.CONSULTANT., 'Engineering', $);
+#8 = IFCAPPLICATION(#6, '15.0 Alpha 490', '12d Model', '12d Model');
+#9 = IFCUNITASSIGNMENT((#10, #11, #12, #13));
+#10 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#11 = IFCSIUNIT(*, .AREAUNIT., $, .SQUARE_METRE.);
+#12 = IFCSIUNIT(*, .VOLUMEUNIT., $, .CUBIC_METRE.);
+#13 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#14 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-6, #16, #20);
+#15 = IFCLOCALPLACEMENT($, #16);
+#16 = IFCAXIS2PLACEMENT3D(#17, #18, #19);
+#17 = IFCCARTESIANPOINT((0., 0., 0.));
+#18 = IFCDIRECTION((0., 0., 1.));
+#19 = IFCDIRECTION((1., 0., 0.));
+#20 = IFCDIRECTION((0., 1.));
+#21 = IFCSITE('2qKSUNpy55uBfNLykvGCun', #2, 'Default Site', 'Description of Default Site', $, #15, $, $, .ELEMENT., $, $, $, $, $);
+#22 = IFCRELAGGREGATES('1Ujh0H4o5Di8enQDkM1fCz', #2, 'ProjectContainer', 'ProjectContainer for Sites', #1, (#21));
+#23 = IFCFACILITY('3tNId_oVX9NhXa_2n$0_rq', #2, 'SA  no VG', 'Description of SA  no VG', $, #15, $, $, .ELEMENT.);
+#24 = IFCRELAGGREGATES('3eZRrNuEDFaPnVCIRe1M4w', #2, 'SiteContainer', 'SiteContainer For All Models', #21, (#23));
+#25 = IFCRELCONTAINEDINSPATIALSTRUCTURE('2fl_Qy6DDBhQbo6E5iWbOW', #2, 'FacilityContainer', 'FacilityContainer for Elements', (#26), #23);
+#26 = IFCALIGNMENT('06CwQ1pbT8Nv9pM2sdAsYi', #2, 'RS01 no VG', '12d Alignment String', 'clothoid', #15, $, #10001, $);
+#10001 = IFCPOLYLINE((#10002,#10003));
+#10002 = IFCCARTESIANPOINT((0., 0., 0.));
+#10003 = IFCCARTESIANPOINT((0., 1., 0.));
+#27 = IFCRELAGGREGATES('2JayEtMO99GeTqChDFMMaF', #2, '', '', #26, (#28));
+#28 = IFCALIGNMENTHORIZONTAL('2KbGXTU9H2TAVcktWH7HXq', #2, 'Horizontal Alignment', '12d Horizontal Alignment', $, $, $, 0., (#29, #31, #33));
+#29 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #30, 0., 0., 0., 48.5550856436632, $, .LINE.);
+#30 = IFCCARTESIANPOINT((0., 0.));
+#31 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #32, 0., 150., 150., 99.118453223473, $, .CIRCULARARC.);
+#32 = IFCCARTESIANPOINT((48.5550856436632, 0.));
```

Comment:
This is not documented very well, but the way I read the [documentation](https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC2/HTML/link/ifcalignmenthorizontalsegment.htm), the radii should be negative here as well.

There is definitely a missing explanation what a negative value for radii means (CCW or !CCW). @SergejMuhic, opinion?

Reply:
I retract my comment. CCW = positive, !CCW = negative.